### DISCUSSION TITLE
S3 alternative(s)?

### DISCUSSION THREAD

**Discussion by rduivenvoorde** [opengisch/QField#2451] [Q&A]
Hi OpenGIS, thanks for OpenSourcing this!

I'm planning to do a local install, so this question is a little premature,  but I wonder if you can do without the S3 ( I'm not allowed to use amazon stuff ;-) ).

I googled, but I cannot find  some FOSS alternative for it. Are you maybe aware of something like that? 
Or is "(Red Hat) Ceph" somethink like that too? I'm pretty green in these 'Simple Storage Solutions'.

Is it only used for image or data uploads? Of is there also other data there?

**Comment by mbernasocchi** [comment]
hi @rduivenvoorde S3 is not only on Amazon. there are plenty of S3 compatibles offering out there. or you can roll out your own with for example min.io

**Comment by rduivenvoorde** [comment]
Ah, sorry for my ignorance; thanks for the pointer. When I googled S3 I really only found Amazon... but that can be good marketing. :-)

Will have a look at min.io

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
